arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -225,12 +225,12 @@
             "$ref": "#/parameters/RegistryNameParameter"
           },
           {
-            "name": "registryUpdateParameters",
+            "name": "registry",
             "in": "body",
             "description": "The parameters for updating a container registry.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/RegistryUpdateParameters"
+              "$ref": "#/definitions/Registry"
```

Comment:
If you allow PATCH for properties other than just tags it is recommended that you define a separate PATCH operation (Resource_Update), else Resource_CreateOrUpdate should suffice.